### PR TITLE
pin dependencies to major version

### PR DESCRIPTION
**Pull request by davidzhanghp**

### CODE REVIEW

File: go/server.go [hunk 1]
```diff
@@ -315,6 +315,10 @@ func linkTokenCreate(
 	return resp.LinkToken, nil
 }
 
+func assets(c *gin.Context) {
```
Comment (otherchen) [inline go/server.go:318]: 👍

File: python/requirements.txt [hunk 2]
```diff
@@ -6,4 +6,4 @@ click==6.7
 itsdangerous==0.24
 requests==2.20.0
 wsgiref==0.1.2 ; python_version < '3.2'
-plaid-python>=3.0.0
+plaid-python>=5.0.0, < 6.0.0
```
Comment (otherchen) [inline python/requirements.txt:9]: why not >= `6.0.0` here?

Comment (davidzhanghp) [reply python/requirements.txt:9]: image

Comment (otherchen) [reply python/requirements.txt:9]: LOL noted

**Pull request merged**